NTumbleBit/ClassicTumbler/Client/TumblerClientRuntime.cs

```diff
@@ -70,7 +70,7 @@ public static TumblerClientRuntime FromConfiguration(TumblerClientConfigurationB
                     await SetupTorAsync(interaction, configuration.TorPath).ConfigureAwait(false);
                 else if (configuration.TorMandatory)
                     throw new ConfigException("The tumbler server should use TOR");
-                var client = CreateTumblerClient(0);
+                var client = CreateTumblerClient(0, connectTimeout: TimeSpan.FromSeconds(5));
```

Comment:
5 seconds may not be sufficient time - in my experience a first Tor connection can be quite slow